src/lib/hooks/use-best-quote/utils/quote-results.ts

```diff
@@ -0,0 +1,89 @@
+import { Token } from '@/constants/tokens'
+import {
+  isAvailableForFlashMint,
+  isAvailableForRedemption,
+  isAvailableForSwap,
+} from '@/lib/utils/tokens'
+
+import { Quote, QuoteResults, QuoteType, ZeroExQuote } from '../types'
+
+import { getBestQuote } from './best-quote'
+
+export function getQuoteResults(
```

Comment:
In a follow up PR I will add `chainId` here and then also in this function test that a quote is only added if `quote.chainId === currentChainId`